src/tests/suite/daos_test_common.c

```diff
@@ -209,21 +211,26 @@ test_setup_cont_create(void **state, daos_prop_t *co_prop)
 	int rc = 0;
 
 	if (arg->myrank == 0) {
-		uuid_generate(arg->co_uuid);
-		print_message("setup: creating container "DF_UUIDF"\n",
-			      DP_UUID(arg->co_uuid));
-		rc = daos_cont_create(arg->pool.poh, arg->co_uuid, co_prop,
+		print_message("setup: creating container\n");
+		rc = daos_cont_create(arg->pool.poh, &arg->co_uuid, co_prop,
```

Comment:
same here, maybe better to use cont_create_with_label to promote that as the better way to play with containers?